packages/@aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts

```diff
@@ -124,7 +125,7 @@ export interface ApplicationLoadBalancedServiceBaseProps {
    *
    * @default - a new load balancer will be created.
    */
-  readonly loadBalancer?: ApplicationLoadBalancer;
+  readonly loadBalancer?: IApplicationLoadBalancer;
```

Comment:
Can you add a comment here that the VPC attribute of the imported load balancer needs to be specified in order to be used?

Reply:
done--is this what you had in mind?